package.json

```diff
@@ -15,12 +15,14 @@
     "build": "tsc --build tsconfig.json",
     "prepare": "yarn build",
     "pretest": "yarn lint",
-    "test": "mocha --timeout 60000 --require ts-node/register --exit test/*.ts test/**/*.ts",
-    "e2e-test": "mocha --timeout 300000 --require ts-node/register --exit e2e-test/**/*.ts",
+    "test": "mocha --timeout 60000 --require ts-node/register --exit test/*.ts",
+    "test:integration": "mocha --timeout 60000 --require ts-node/register --exit test/integration/*.ts",
     "start": "node ./dist/index.js start"
   },
   "dependencies": {
-    "@polkadot/api": "^12.0.1",
+    "@polkadot/api": "12.0.1",
```

Comment:
can we bump it to the latest ? https://github.com/polkadot-js/api/releases/tag/v12.2.3

Reply:
Yes, we can. I choose not to do it initially to minimize the changes needed for merging this PR. Otherwise I think, we would also need to update `@w3f/polkadot-api-client` and `@w3f/test-utils`.